lib/resin-lint.coffee

```diff
@@ -8,14 +10,26 @@ reporter = require('coffeelint/lib/reporters/default')
 
 CONFIG_PATH = path.join(__dirname, '../config/coffeelint.json')
 
+# The linter expects the path to actual source files, for example:
+#		src/
+#		test/
+#	but depcheck expects the root of a project directory (where the
+#	package.json is). This function takes a path and propagates upwards
+#	until it contains a package.json
+getPackageJsonDir = (dir) ->
```

Comment:
The interface would still be the same, it'd just be that you have to be explicit in the path to the package.json to test, rather than search up the dir tree to find it